sql/moz-fx-data-shared-prod/cloudflare/browser_usage/view.sql

```diff
@@ -2,6 +2,23 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.cloudflare.browser_usage`
 AS
 SELECT
-  *
+  brwsr_usg.*,
+  country_codes.location_name,
+  country_codes.region_name
 FROM
-  `moz-fx-data-shared-prod.cloudflare_derived.browser_usage_v1`
+  `moz-fx-data-shared-prod.cloudflare_derived.browser_usage_v1` brwsr_usg
+LEFT JOIN
+  (
+    SELECT
+      'ALL' AS code,
+      'Global' AS location_name,
```

Comment:
Do we call countries `location_name` anywhere else?

Reply:
I totally get what you're saying but because this dropdown includes "global" as an option, I wanted to make it more generic